openedx/features/enterprise_support/tests/factories.py

```diff
@@ -71,6 +71,5 @@ class Meta(object):
 
         model = EnterpriseCourseEnrollment
 
-    id = factory.LazyAttribute(lambda x: FAKER.random_int(min=1))  # pylint: disable=no-member
```

Comment:
`FAKER.random_int(min=1)` can generate redundant numbers.
we do not need to fake `id` as it is an auto-generated field.